oak-security-spi/src/main/java/org/apache/jackrabbit/oak/spi/security/user/UserConstants.java

```diff
@@ -78,6 +78,11 @@ public interface UserConstants {
      */
     String PARAM_ADMIN_ID = "adminId";
 
+    /**
+     * Configuration option defining the ID of the impersonatorGroups field.
+     */
+    String PARAM_IMPERSONATOR_GROUPS_ID = "impersonatorGroups";
```

Comment:
the parameter should be based on principal names and not on IDs.
see discussion above about the 2 options:
- we define that it must contain names of group principals => only change the ID part
- we define that i may contain any principal name => drop group and replace ID>